### DISCUSSION TITLE
API to fetch JSON Schema of the content type

### DISCUSSION THREAD

**Discussion by manslogic** [directus/directus#16600] [Archive]
We will configure content types in directus, instantly Rest API and GraphQL APIs are available, but for now i cannot get the **JSON Schema** of the content type. it would be worth to have the JSON Schema API

Let me know is there a way to get the JSON Schema for the defined content type

**Comment by azrikahar** [comment]
You should be able to retrieve OpenAPI specification for your project (which should include the JSON Schema you're looking for): https://docs.directus.io/reference/system/server.html#get-openapi-specification
